seatunnel-api/src/main/java/org/apache/seatunnel/api/serialization/DeserializationSchema.java

```diff
@@ -29,13 +30,15 @@ public interface DeserializationSchema<T> extends Serializable {
      * Deserializes the byte message.
      *
      * @param message The message, as a byte array.
+     * @param tablePath The Registry name.
      * @return The deserialized message as an SeaTunnel Row (null if the message cannot be
      *     deserialized).
      */
-    T deserialize(byte[] message) throws IOException;
+    T deserialize(byte[] message, TablePath tablePath) throws IOException;
```

Comment:
Why add it here, it's pretty destructive

cc @Hisoka-X @EricJoy2048

Reply:
> Why add it here, it's pretty destructive
> 
> cc @Hisoka-X @EricJoy2048

You mean add another implementation?

Reply:
Yes, I suggest add new function.

Reply:
> Yes, I suggest add new function.
done